Acala/acala-evm-starter/docker/pg-Dockerfile

```diff
@@ -6,4 +6,7 @@ ENV POSTGRES_USER 'postgres'
 ENV POSTGRES_PASSWORD 'postgres'
 
 # Copy in the load-extensions script
-COPY docker/load-extensions.sh /docker-entrypoint-initdb.d/
\ No newline at end of file
+COPY docker/load-extensions.sh /docker-entrypoint-initdb.d/
+
+# Convert line endings to LF
+RUN sed -i 's/\r$//' /docker-entrypoint-initdb.d/load-extensions.sh && chmod +x /docker-entrypoint-initdb.d/load-extensions.sh
```

Comment:
Do we need to script this? Or can we just update `docker/load-extensions.sh` with the correct line endings and commit the updated file?

Reply:
the handling of line endings seems to be largely determined by the environment itself. Changing CRLF to LF is not being considered a change that can be committed. Also, manually adding the line feed character to the end of each line is only considered a literal. But, the Docker COPY command seems to preserve the line endings as it is in the host. So, I think the sed command is necessary.